src/test/resources/gold/formatting/dlanguage_issue_497-after.d

```diff
@@ -1,7 +1,7 @@
 import std.stdio;
 
 void main(string[] args){
-    switch(i){
+    switch (i){
```

Comment:
perhaps it should enforce a space before the opening brace as well

Reply:
Yeah, right now it just keeps whatever you type, but it'd be cool if we enforced consistent spacing. The Java formatter has a bunch of options for spacing braces
<img width="345" alt="image" src="https://user-images.githubusercontent.com/1817241/113902473-c619f800-9784-11eb-90a6-cc5812070eaf.png">

I've been thinking of maybe trying to do some more settings-driven formatter stuff (maybe a bit of refactoring in the formatter is warranted). I can look into spaces before the left brace. Would you prefer to receive a more complete PR, or smaller pieces?

Reply:
have merged this for now but would certainly welcome additional work in this area, or any area in fact.